### ISSUE TITLE
Display current score difference from high score on breaks

### ISSUE THREAD

**Issue by inkfarer** [ppy/osu#12657]
As a player, I frequently wonder how I am doing compared to my high score while playing. Showing how the player's current play session is doing compared to their high score at that point on the beatmap would be an useful way for players to see how likely they are to improve their high score on the current session.

A mockup is attached below.

![score-difference-mockup](https://user-images.githubusercontent.com/6833028/116810645-9d034200-ab4d-11eb-9408-c52ccfd67116.png)

**Comment by bdach** [comment]
I would probably rather have something like the mod proposed in #6642 rather than add this. I certainly wouldn't like to see this every break when I'm playing for fun.

**Comment by frenzibyte** [comment]
Yeah I don't really agree with this as well, nor I see the point of it, and as has been mentioned above, a "beat score" mod (or something else rather than a mod?) would probably do far better.

**Comment by eclecticfluff** [comment]
Considering how the skin element GUI is being worked on, could the layout of information on breaks be combined so it’s modifiable by the same method? It might not be as simple as a toggle somewhere, but the flexibility to add any number of potential statistics, and restructure the information would allow for even more changes. Though having to go through the skin layout interface may well come at the cost of intuitiveness, so a simple toggle could be the best design.

**Comment by peppy** [comment]
This would definitely be fine added as proposed. We can deal with requests to hide such information at a later point in time (via more customisation over what is displayed on the break screen).
